engine/src/cmd/unit_factory.h

```diff
@@ -123,94 +118,11 @@ class UnitFactory
                                       int faction,
                                       Flightgroup *fg = NULL,
                                       int fg_snumber = 0,
-                                      float difficulty = .01,
-                                      ObjSerial netcreate = 0 );
+                                      float difficulty = .01 );
```

Comment:
float values should be `0.01f`... IIRC `0.01` would be a double getting converted to a float.